from errorcatch import error_handle_test

# This module checks whether this can be successfully configured pre-use
error_handler = error_handle_test("This is the message")
